src/sgame/sg_cmds.cpp

```diff
@@ -3497,6 +3497,98 @@ void Cmd_Reload_f( gentity_t *ent )
 	}
 }
 
+/*
```

Comment:
Those "headers" are just really useless (they are even annoying when solving git conflicts, as sequences of `=` are used to show where the conflict goes to the 2nd side of the problem), we don't do them anymore, instead we write proper documentation for the functions.